docs/object.md

```diff
@@ -26,7 +26,7 @@ type car('a) = {
 } as 'a;
 ```
 
-Two dots, also called an elision, indicate that this is an "open" object type, and therefore can also contain other values and methods. An open object is also polymorphic and therefore requires a parameter.
+Two dots, also called an elision, indicate that this is an "open" object type, and therefore can also contain other values and methods. An open object is also polymorphic and therefore requires a parameter. This parameter refers to the complete type of the object, including those other values and methods.
 
```

Comment:
Agree here with you that this isn't right, since open object is the form of structural typing.

I'm not sure if this entire paragraph make sense after your addition thought. Since you refer to the `as 'a` without being very explicit.

Reply:
Just checking: By "Agree with you that this isn't right", are you referring to the old code example, rather than the part that is quoted here?

I don't see how the sentence I added would cause the paragraph to make less sense than it did previously. It already referred to the `'a` when it said "and therefore requires a parameter".